Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -1,12 +1,26 @@
 # Amazon.Lambda.Annotations
 
-The Lambda Annotations is a programming model for writing .NET Lambda function. At a high level the programming model allows
-idiomatic .NET coding patterns and uses [C# Source Generators](https://docs.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/source-generators-overview) to bridge the gap between the Lambda programming model 
-to the more idiomatic programming model.
-
-For example here is a simplistic example of a .NET Lambda function that acts like a calculator plus method using the normal 
-Lambda programming model. It respondes to an API Gateway REST API, pulls the operands from the resource paths, does the 
+The Lambda Annotations is a programming model for writing .NET Lambda functions. At a high level the programming model allows
```

Comment:
nit: agreement feels off here, I'd probably just drop the first "The"

Thought about "...annotations _are_...", but fine if it's a singular "model"

Reply:
Removed the `The`